.ci-builds/.azure-pipelines.yml

```diff
@@ -703,7 +703,19 @@ jobs:
         mkdir -p icuexportdata/collation/implicithan
         ./bin/genrb -X -s data/coll/ --ucadata data/in/coll/ucadata-implicithan-icu4x.icu -d icuexportdata/collation/implicithan $FILES
         rm icuexportdata/collation/implicithan/*.res
-      displayName: 'Build collation data export files'
+      displayName: 'Build collation data files'
+      env:
+        LD_LIBRARY_PATH: lib
+    - script: |
+        cd icu4c/source
+        mkdir -p icuexportdata/segmenter/dictionary
+        for FILE in `ls data/brkitr/dictionaries`
+        do
+          ./bin/gendict --uchars data/brkitr/dictionaries/$FILE /dev/stdout | \
```

Comment:
should this be a part of the icuexportdata binary? It would be nice if everything that belongs in icuexportdata.zip is generatable by the icuexportdata binary


plus i don't like splicing in python that only exists on CI

Reply:
Collation data is also not through the binary.

Why write and maintain more code if we already have a tool for it?

Reply:
It's not more code, it's including the existing code in a different tool, no?

Basically call these tools as a library.

Reply:
I don't know how to do that and don't really want to spend a day learning how ICU works.

Reply:
Yeah I was going to comment on this but if it works it works and it's not a great use of time to plumb it in with the actual icuexportdata binary especially given that this boat already sailed with the collator data.

One small improvement though would be to take this inline python code and make it a .py file to make it easier to reproduce locally.

Reply:
Can I get this merged an do it in a follow-up?

Reply:
Sure but perhaps in the same `maint` uplift cycle.